lib/utils/utils.dart

```diff
@@ -21,3 +26,20 @@ class _TimestampFromStringJsonConverter
   String toJson(DateTime object) =>
       dateSecondsSinceEpochJsonConverter.toJson(object).toString();
 }
+
+class NtpTime {
+  NtpTime._();
+
+  /// Returns current time of the system clock with offset from the NTP server
+  /// Use this method instead of [DateTime.now] to get the correct time
+  static DateTime now() => inject<NtpService>().now();
+
+  static DateTime clockNow() =>
+      // ignore: avoid_datetime_now
+      Platform.environment.containsKey('FLUTTER_TEST') ? clock.now() : now();
```

Comment:
Please, leave comment why we need check for FLUTTER_TEST

Reply:
Sure 🤟